indico/core/__init__.py

```diff
@@ -19,21 +19,17 @@
 from indico.core import signals
 from indico.core.config import Config
 from indico.util.i18n import _
-from indico.web.flask.util import url_for
 from indico.web.menu import TopMenuSection, TopMenuItem
 from indico.web.util import url_for_index
 
+__all__ = ('Config',)
```

Comment:
do we ever import `Config` from `indico.core` instead of `indico.core.config`?

Reply:
Yeah, I saw it somewhere. But yeah, we can change that.